lib/main.coffee

```diff
@@ -46,6 +46,16 @@ module.exports =
       lintOnFly: true
       lint: (textEditor) =>
         filePath = textEditor.getPath()
+        # check if project dir contains .coafile
+        check = helpers.exec(@executable,
+                             ['--find-config'],
+                             {cwd: path.dirname(filePath)}).then (result) ->
+          atom.notifications.addSuccess('Check completed').catch (error) ->
+            atom.notifications.addInfo \
+              'Deactivating coala!',
```

Comment:
I take it you've tested it locally by running on `atom -d`? 
I'll try it out myself, and write tests for checking it later.
It looks good, if its works then the issue is resolved :)